Decompiler/Options.cs

```diff
@@ -42,6 +42,18 @@ class Options
             HelpText = "Verify checksums and signatures.")]
         public bool VerifyVPKChecksums { get; set; }
 
+        [Option('d', "vpk_dump",
```

Comment:
Well, it's called decompiler for a reason :) 
But since dumping comes first, I guess that would work too.
Will do a test and update on results.